src/Platform/MysqlPlatform.php

```diff
@@ -150,14 +150,12 @@ protected function createDropTriggersDDL(Table $table)
      */
     protected function getTriggerNames(Table $table)
     {
-        $tableName = $table->getName();
-
         /** @var CompositeNumberRangeBehavior $behavior */
         $behavior = $table->getBehavior(self::BEHAVIOR_NAME);
-        $foreignTableName = $behavior->getForeignTable();
+        $compositeKeyColumnName = $behavior->getCompositeKeyColumnName();
 
-        $insertTrigger = str_replace(' ', '', ucwords(str_replace('_', ' ', 'set' . ucfirst($foreignTableName) . ucfirst($tableName) . 'Id')));
-        $updateTrigger = str_replace(' ', '', ucwords(str_replace('_', ' ', 'setOnUpdate' . ucfirst($foreignTableName) . ucfirst($tableName) . 'Id')));
+        $insertTrigger = str_replace(' ', '', ucwords(str_replace('_', ' ', 'set' . ucfirst($compositeKeyColumnName))));
```

Comment:
❓ Wäre hier nicht ein potenzieller breaking change? Es würde ein ucfirst auf dem tableName nicht mehr gemacht werden, da es in der Methode getCompositeKeyColumnName nicht vorgesehen ist.